provider/resources.go

```diff
@@ -399,6 +217,8 @@ func Provider() tfbridge.ProviderInfo {
 		},
 	}
 
+	tfbridge.MustTraverseProperties(&prov, "ids", applyResourceIDs)
```

Comment:
Nifty, I was sure there must be a more general way to apply the ids, but wasn't sure what the right helper method would be!